### PR TITLE
Update scoring_functions.py

### PR DESCRIPTION
**Pull request by tanujdhiman**
We can find accuracy by manually if we are not going to use sklearn library.

### CODE REVIEW

**Comment by cclauss** [comment]
https://travis-ci.com/github/TheAlgorithms/Python/builds/178988952#L252

**Comment by spamegg1** [comment]
@tanujdhiman All you need to do is remove whitespaces from lines 143 and 145

**Comment by tanujdhiman** [comment]
@spamegg1 I think it is a good practice of not dependent on some library like Sklearn. It's my suggestion. What are you views ?

File: machine_learning/scoring_functions.py [hunk 1]
```diff
@@ -135,3 +135,10 @@ def mbd(predict, actual):
     score = float(numerator) / denumerator * 100
 
     return score
+
+
+def manual_accuracy(predict, actual):
+    predict = np.array(predict)
+    actual = np.array(actual)
+    score = np.mean(actual == predict)
+    return score
```
Comment (cclauss) [inline machine_learning/scoring_functions.py:144]:
```suggestion
    return np.mean(np.array(actual) == np.array(predict))
```

**Review by cclauss — CHANGES_REQUESTED** [review]

File: machine_learning/scoring_functions.py (hunk 1 above)
Comment (tanujdhiman) [reply machine_learning/scoring_functions.py:144]: Yep we can do this by just writing a single line.
Thanks @cclauss

**Review by cclauss — APPROVED** [review]

**Review by tanujdhiman — COMMENTED** [review]
Sure.

**Pull request merged**